src/index.js

```diff
@@ -0,0 +1,362 @@
+import data from './js/Data';
+import Key from './js/Key';
+
+const unpressShift = () => {
+  document.querySelectorAll('.symbol_unshift').forEach((symbol) => {
+    symbol.classList.remove('hide');
+  });
+  document.querySelectorAll('.symbol_shift').forEach((symbol) => {
+    symbol.classList.add('hide');
+  });
+};
+const pressShift = () => {
+  document.querySelectorAll('.symbol_unshift').forEach((symbol) => {
+    symbol.classList.add('hide');
+  });
+  document.querySelectorAll('.symbol_shift').forEach((symbol) => {
+    symbol.classList.remove('hide');
+  });
+};
+
+// render wrapper
+const renderWrapper = () => {
+  const wrapper = document.createElement('div');
+  wrapper.className = 'wrapper';
+  document.body.append(wrapper);
+  return wrapper;
+};
+// textarea rendering
+const renderTextarea = (wrapper) => {
+  const textarea = document.createElement('textarea');
+  textarea.id = 'textarea';
+  textarea.setAttribute('rows', '15');
+  wrapper.append(textarea);
+};
+// Keyboard rendering
+const renderRows = (dataKeyboard) => {
+  if (!localStorage.getItem('language')) {
+    localStorage.setItem('language', 'eng');
+  }
+  const dataKeys = dataKeyboard[localStorage.getItem('language')];
+  const rows = [];
+  dataKeys.forEach((dataRow) => {
+    const row = document.createElement('section');
+    row.className = 'row';
+
+    dataRow.forEach((dataKey) => {
+      const key = new Key(dataKey);
+      const button = key.generateKey();
+      row.append(button);
+    });
+    rows.push(row);
+  });
+  return rows;
+};
+const renderKeyBoard = () => {
+  const keyboard = document.createElement('section');
+  keyboard.className = 'keyboard';
+
+  const rows = renderRows(data);
+  rows.forEach((row) => {
+    keyboard.append(row);
+  });
+  return keyboard;
+};
+//---------------------------------------------
+// change Language
+const changeLanguage = (e) => {
+  e.preventDefault();
+  if (localStorage.getItem('language') === 'eng') {
```

Comment:
It is OK for two languages, but if you have 100 there will be lots of `if` states. You can create create a constant, for example 
`nextKeyboardLanguage = {
"eng" : "ru",
"ru": "eng"
}
`
You will store the next language here and instead of lines 69-73 use `localStorage.setItem(nextKeyboardLanguage[localStorage.getItem('language')])`